giddy/markov.py

```diff
@@ -550,7 +551,7 @@ class Spatial_Markov(object):
     Let's still use the US state income time series to demonstrate. We first
     discretize them into categories and then pass them to Spatial_Markov.
 
-    >>> import mapclassify.api as mc
+    >>> import mapclassify as mc
```

Comment:
the current pypi release of `mapclassify` is in August 2017 which still adopted the `api.py` strategy. Once we have a new release on pypi, the travis test should pass.